README.md

```diff
@@ -31,7 +31,16 @@ swap(one, -1, 0)
 console.log(one) // [1]
 
 function swap(array, first_index, second_index) {
-  // -- YOUR ANSWER HERE --
+  if(first_index === undefined){
+    console.log("Your first index is invalid");
+  } else if (second_index === undefined) {
+    console.log("Your second index is invalid");
+  } else {
+    let tmp = array[first_index];
+    let tmp2 = array[second_index]
+    array[first_index] = array[second_index];
+    array[second_index] = array[first_index];
```

Comment:
This is not going to give you the result you are aiming for.  Please test, and then fix.  Also, you just need one temporary variable.